doc/source/index.md

```diff
@@ -2,12 +2,11 @@
    :hidden:
 
    Home <self>
+   API <api>
```

Comment:
Why this change?

Reply:
good question, that shouldn't have happened 🤔